Dockerfile

```diff
@@ -1,24 +1,30 @@
-FROM centos:8 as base
+FROM quay.io/centos/centos:stream9 as base
 ARG channel="stable"
 ARG location
 
 RUN [ -z "${channel}" ] && echo "ARG channel is required" && exit 1 || true
 
-RUN yum -y install jq
-RUN curl https://builds.coreos.fedoraproject.org/streams/${channel}.json -o stable.json && \
-	cat stable.json | jq '.architectures.x86_64.artifacts.qemu.release' | tr -d '"'
+RUN yum -y install jq xz
+RUN ARCH=$(uname -m) ; echo $ARCH \
+    ; if [ "$ARCH" == "s390x" ] ; then ARCH=s390x ; elif [ "$ARCH" == "x86_64" ] ; then ARCH=x86_64 ; fi \
+    ; curl https://builds.coreos.fedoraproject.org/streams/${channel}.json -o stable.json && \
+        cat stable.json | jq --arg arch "$ARCH" '.architectures[$arch].artifacts.qemu.release' | tr -d '"' 
```

Comment:
instead of `tr -d '"'` you can run jq with -r

Reply:
Sure. Will use the `jq -r` instead of `tr`

Thanks for the tip ✌️